bfillings/mothur.py

```diff
@@ -496,7 +512,7 @@ def parse_mothur_assignments(lines):
 
 def mothur_classify_file(
         query_file, ref_fp, tax_fp, cutoff=None, iters=None, ksize=None,
-        output_fp=None):
+        output_fp=None, tmp_dir="/tmp"):
```

Comment:
@kylebittinger, this shouldn't be ``/tmp`` here, right? It also looks like ``/tmp`` is still hard-coded in other places in this file - can you do a find/replace for those?